kubernetes-maven-plugin/doc/src/main/asciidoc/inc/goals/develop/_jkube-log.adoc

```diff
@@ -39,6 +39,31 @@ If your pod has multiple containers you can configure the container name to log
 mvn {goal-prefix}:log -Djkube.log.container=foo
 ----
 
+.Example logs configuration
```

Comment:
```suggestion
.Example XML configuration for log goal
```

Reply:
Changed to `.Example XML configuration for log goal`